app/application/controllers/app.rb

```diff
@@ -18,6 +19,104 @@ class App < Roda
       routing.public # make GET public/images/
       response['Content-Type'] = 'application/json'
 
+      # routing.on 'init' do
+      #   # posts = Repository::For.klass(Entity::Posts).all
+      #   NoFB::Database::UsersOrm.find_or_create(user_id: '123',
+      #                                     user_email: 'hhoracehsu@@gmail.com',
+      #                                     user_name: '242234',
+      #                                     user_img: 'img test')
+
+      #   NoFB::Database::GroupsOrm.find_or_create(group_id: '4534',
+      #                                      group_name: 'lalalala')
+
+      #   NoFB::Database::GroupsOrm.find_or_create(group_id: '8787',
+      #                                      group_name: 'Idiot')
+
+      #   NoFB::Database::SubscribesOrm.create(user_id: '123',
+      #                                  group_id: '4534',
+      #                                  word: 'cookies, sell')
+      #   NoFB::Database::SubscribesOrm.create(user_id: '123',
+      #                                  group_id: '8787',
+      #                                  word: 'Bread')
+      # end
+
```

Comment:
Not sure this part for?

Reply:
Sorry for the late reply. This part is for debugging. We have updated for a deleted version.